src/logic.py

```diff
@@ -176,6 +177,25 @@ def assign_author(self) -> tuple[bool, str]:
             True,
             f'Author {self.author} successfully assigned to PR, "{self.config.awaiting_update_label}" label added',
         )
+    
+    def parse_magic_string(self) -> str:
+        """
+            Parses the commit body to find magic string
```

Comment:
```suggestion
            Parses the commit body to find magic string
```
Please fix the indent, we should be looking for the magic string in the PR body, not the commit message.